stdlib/linecache.pyi

```diff
@@ -7,7 +7,7 @@ else:
     __all__ = ["getline", "clearcache", "checkcache"]
 
 _ModuleGlobals = dict[str, Any]
-_ModuleMetadata = tuple[int, float, list[str], str]
+_ModuleMetadata = tuple[int, float, list[str] | None, str]
```

Comment:
Isn't `mtime` the second item in the tuple, rather than the third item?

https://github.com/python/cpython/blob/3.10/Lib/linecache.py#L68

Reply:
Crap. You're right